Weather/Views/CityNameView.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CityNameView.swift
+//  Weather
+//
+//  Created by Alex on 5.08.22.
+//
+
+import SwiftUI
+
+struct CityNameView: View {
+    var city: String
+    var date: String
+    
+    var body: some View {
+        HStack {
+            VStack(alignment: .center, spacing: 10) {
```

Comment:
These parameters are default. You can just write: `VStack {`

Reply:
Removed.)